manifests/interface.pp

```diff
@@ -40,6 +43,7 @@
 define wireguard::interface (
   String[1] $public_key,
   Optional[String[1]] $endpoint = undef,
+  Integer[0, 65535] $persistent_keepalive = 0,
```

Comment:
we can add the parameter with  a default of 0 because that's the same upstream. So not a breaking change.